src/v/kafka/server/group.cc

```diff
@@ -2244,14 +2252,16 @@ group::handle_prepare_tx(cluster::prepare_group_tx_request r) {
 }
 
 ss::future<cluster::abort_group_tx_reply>
-group::handle_abort_tx(cluster::abort_group_tx_request r) {
+group::handle_abort_tx(cluster::abort_group_tx_request req) {
     if (in_state(group_state::dead)) {
         cluster::abort_group_tx_reply reply;
         reply.ec = cluster::tx_errc::coordinator_not_available;
         co_return reply;
     } else if (
       in_state(group_state::stable) || in_state(group_state::empty)
       || in_state(group_state::preparing_rebalance)) {
+        auto r = std::move(req);
```

Comment:
this looks unnecessary. is there something i'm missing? same for the other cases where the parameter is taken by value and you move it on to the stack.

Reply:
Yes, my fault. Will delete it